components/HeroUnit.js

```diff
@@ -52,6 +53,7 @@ function HeroUnitLogo () {
     <div>
       <div className='logo'><ReactBKKLogo /></div>
       <h1>React Bangkok</h1>
+      <Countdown date={'03 June 2017 09:30:00'} renderer={renderer} />
```

Comment:
Can you defer the rendering of a countdown until the page is mounted on the client? Otherwise, when running, React will fail due to DOM content mismatch:

![image](https://cloud.githubusercontent.com/assets/193136/26275833/667483c8-3d94-11e7-92e5-8905752b2386.png)

A solution is to create a container component that only renders the countdown when it is mounted on the client. Like this:

```js
class CountdownContainer extends React.Component {
  state = {
    mounted: false
  }
  componentDidMount () {
    this.setState({ mounted: true })
  }
  render () {
    if (this.state.mounted) {
      return <Countdown date={'03 June 2017 09:30:00'} renderer={renderer} />
    } else {
      return renderer()
    }
  }
}